services/filemanager/filemanager.go

```diff
@@ -101,7 +120,6 @@ func GetProviderConfigForBackupsFromEnv(ctx context.Context) map[string]interfac
 		providerConfig["regionHint"] = config.GetEnv("AWS_S3_REGION_HINT", "us-east-1")
 		providerConfig["iamRoleArn"] = config.GetEnv("BACKUP_IAM_ROLE_ARN", "")
 		if providerConfig["iamRoleArn"] != "" {
-			backendconfig.DefaultBackendConfig.WaitForConfig(ctx)
```

Comment:
What if in the future this method is called when backendConfig is not ready?

Reply:
we could inline both methods in the calling method (`GetBackupStorageConfig`)